module/main.js

```diff
@@ -2,21 +2,35 @@
 
 // Import Modules
 import { preloadHandlebarsTemplates } from './templates.js'
+import { migrateWorld } from './migration.js';
 import { VampireActor } from './actor/actor.js'
-import { VampireActorSheet, rollDice } from './actor/actor-sheet.js'
 import { VampireItem } from './item/item.js'
 import { VampireItemSheet } from './item/item-sheet.js'
 import { VampireDie, VampireHungerDie } from './dice/dice.js'
+import { rollDice } from './actor/roll-dice.js'
+import { CoterieActorSheet } from './actor/coterie-actor-sheet.js';
+import { MortalActorSheet } from './actor/mortal-actor-sheet.js'
+import { GhoulActorSheet } from './actor/ghoul-actor-sheet.js'
+import { VampireActorSheet } from './actor/vampire-actor-sheet.js'
 
 Hooks.once('init', async function () {
   console.log('Initializing Schrecknet...')
 
+  game.settings.register('vtm5e', 'worldVersion', {
```

Comment:
I'm trying to insert a migration logic system.

We set a value to current version (1.5) and when that version changes... we can update.